prerendering.bs

```diff
@@ -658,6 +673,33 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+[=Accept-CH cache=], which is owned by user agent as a global storage, should not be modified by [=prerendering browsing context=].
```

Comment:
I don't understand what this means as normative text. Is it merely informative text to explain the following?

Reply:
I meant to say that prerender browsing context should not modify Accept-CH cache, so it makes changes to its own cache before being activated.

Should I include this information in normatice text? I removed it for now.

Reply:
I think it's useful to state this as a [statement of fact](https://ln.hixie.ch/?start=1140242962&count=1), but indeed, stating it as a normative requirement using "must" and "should not" is not correct. So something like

> We need to ensure that the Accept-CH cache, which his owned..., is not modified while prerendering, but is properly updated after activation. The following modifications ensure this:

Reply:
Revised. Thanks! 
(Thanks for the link! Now I understnad why I should not use "should" here :)) )